jdock/src/main/java/io/quarkus/images/modules/GraalVMModule.java

```diff
@@ -25,8 +25,8 @@ public class GraalVMModule extends AbstractModule {
               && rm -Rf %s""";
 
     private static final String NEW_TEMPLATE = """
-            tar xzf %s -C /opt \\
-              && mv /opt/graalvm-community-openjdk-%s* /opt/graalvm \\
+            mkdir -p /opt/graalvm \\
+              && tar xzf %s -C /opt/graalvm --strip-components=1 \\
```

Comment:
I'm pretty sure I did exactly that before...

Reply:
You are right :)

https://github.com/quarkusio/quarkus-images/blob/5460cb23054ce517b25e2a65b72937f80b0eca31/jdock/src/main/java/io/quarkus/images/modules/MandrelModule.java#L16-L19